packages/chart/src/components/LineChart/index.tsx

```diff
@@ -38,16 +38,12 @@ const LineChart = (props: LineChartProps) => {
     } = props
 
   // prettier-ignore
-  const { colorScale, config, formatNumber, handleLineType, parseDate, seriesHighlight, tableData, transformedData, updateConfig, brushConfig,clean  } = useContext<ChartContext>(ConfigContext)
-  const { yScaleRight } = useRightAxis({ config, yMax, data: transformedData, updateConfig })
+  const { colorScale, config, formatNumber, handleLineType, parseDate, seriesHighlight, tableData, transformedData:data, updateConfig,   } = useContext<ChartContext>(ConfigContext)
+  const { yScaleRight } = useRightAxis({ config, yMax, data: data, updateConfig })
```

Comment:
You can just used `data` here over `data: data`

Reply:
Fixed.